_episodes/03-monitor-the-model.md

```diff
@@ -449,110 +454,110 @@ history = model.fit(X_train, y_train,
 ~~~
 {: .language-python}
 
-> ## Exercise: plot the training progress.
->
-> As before the history allows plotting the training progress. But now we can plot both the performance on the training data and on the validation data!
-> * Is there a difference between the training and validation data? And if so, what would this imply?
->
-> > ## Solution
-> > ~~~
-> > history_df = pd.DataFrame.from_dict(history.history)
-> > sns.lineplot(data=history_df[['root_mean_squared_error', 'val_root_mean_squared_error']])
-> > plt.xlabel("epochs")
-> > plt.ylabel("RMSE")
-> > ~~~
-> > {: .language-python}
-> > ![Output of plotting sample](../fig/03_training_history_2_rmse.png)
-> >
-> > This clearly shows that something is not completely right here.
-> > The model predictions on the validation set quickly seem to reach a plateau while the performance on the training set keeps improving.
-> > That is a clear signature of overfitting.
-> {:.solution}
-{:.challenge}
+With this we can plot both the performance on the training data and on the validation data!
+
+~~~
+history_df = pd.DataFrame.from_dict(history.history)
+sns.lineplot(data=history_df[['root_mean_squared_error', 'val_root_mean_squared_error']])
+plt.xlabel("epochs")
+plt.ylabel("RMSE")
+~~~
+{: .language-python}
+![Output of plotting sample](../fig/03_training_history_2_rmse.png)
+
```

Comment:
The exercise here also contained a question that we can still keep in as exercise:

```suggestion
> ## Exercise: plot the training progress.
>
> Is there a difference between the training and validation data? And if so, what would this imply?
>
> > ## Solution
> > This clearly shows that something is not completely right here.
> > The model predictions on the validation set quickly seem to reach a plateau while the performance on the training set keeps improving.
> > That is a clear signature of *overfitting*.
> {:.solution}
{:.challenge}
```

Reply:
This is a nice suggestion. Are you able to remove the occurances of `clearly` (line 472) and `clear` (line 474). I think this demotivates people which do not see it clearly. ;-)

Reply:
```suggestion
> ## Exercise: plot the training progress.
>
> Is there a difference between the training and validation data? And if so, what would this imply?
>
> > ## Solution
> > This shows that something is not completely right here.
> > The model predictions on the validation set quickly seem to reach a plateau while the performance on the training set keeps improving.
> > That is a common signature of *overfitting*.
> {:.solution}
{:.challenge}
```